cirq-google/cirq_google/op_serializer.py

```diff
@@ -30,6 +32,78 @@
 Gate = TypeVar('Gate', bound=ops.Gate)
 
 
+class OpSerializer(abc.ABC):
+    """Generic supertype for op serializers."""
```

Comment:
Same here, recommend using operation.  It might be good to explain each of these abstract classes further, if possible as well.

Reply:
Done.